k3k-kubelet/translate/host.go

```diff
@@ -0,0 +1,105 @@
+package translate
+
+import (
+	"encoding/hex"
+	"fmt"
+
+	"github.com/rancher/k3k/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	// ClusterNameLabel is the key for the label that contains the name of the virtual cluster
+	// this resource was made in
+	ClusterNameLabel = "virt.cattle.io/clusterName"
```

Comment:
are we using `virt.cattle.io` all around the cluster? since we already using k3k.io to the finalizers we add

Reply:
I'm fine with changing the label, I can use "k3k.io/" as the prefix if you would prefer.